Add workaround for marazmista/radeon-profile#151

FMeinicke opened the pull request:
If the PC returns from sleep/hibernation we can enforce restoration of
the correct fan control mode by checking if the correct value is set for
pwm_enable and restore the value if necessary

FMeinicke committed:
Add workaround for marazmista/radeon-profile#151

If the PC returns from sleep/hibernation we can enforce restoration of
the correct fan control mode by checking if the correct value is set for
pwm_enable and restore the value if necessary

marazmista commented on radeon-profile/gpu.cpp:
```diff
@@ -265,12 +265,24 @@ void gpu::setForcePowerLevel(const QString &newForcePowerLevel) {
 
 void gpu::setPwmManualControl(bool manual) {
     driverHandler->setNewValue(getDriverFiles().hwmonAttributes.pwm1_enable, QString(manual ? pwm_manual : pwm_auto));
+    pwmEnabled = manual;
```
Is this variable needed? I think that after resume value will be the same as when system was suspended so it won't check the file. I think just checking the file will be ok.

FMeinicke replied:
Yep, no problem. I'll get rid of this variable then

marazmista commented on radeon-profile/gpu.cpp:
```diff
@@ -265,12 +265,24 @@ void gpu::setForcePowerLevel(const QString &newForcePowerLevel) {
+    // If the PC is sent to sleep (or hibernate) it can happen that PWM is
+    // disabled (by the OS..?) and that we have to re-enable it, if needed,
+    // after returning from sleep
+    static QFile pwmEnableFile(getDriverFiles().hwmonAttributes.pwm1_enable);
```
It can be without static I think

FMeinicke replied:
It could definitely be non-static. But I thought since the file is always the same it would suffice to construct the QFile object once and re-use it after that to gain a bit of performance (because this function is called quite frequently if I'm correct).

But if you prefer to have it non-static it's no problem for me.

FMeinicke committed:
Remove unnecessary variable holding current PWM state

To get around the issue that PWM isn't restored after hibernation it's
enough to check the pwm_enable file - we don't need to also store the
current PWM state into a variable.
